src/Download.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+import fileDownload from 'js-file-download'
+
+export default function Download({ query }) {
+  const [selected, setSelected] = useState('json')
+  const options = ['json', 'csv', 'xlsx', 'tsv', 'ods']
+  const [downloading, setDownloading] = useState('')
+
+  const handleChange = (event) => {
+    setSelected(event.target.value)
+  }
+
+  const downloadData = () => {
+    setDownloading('Preparing Download')
+    axios(`http://localhost:8080/v1/download?format=${selected}`, {
```

Comment:
I'm not sure about using axios as we aren't using it in other places. I think we've used `fetch` but please check how are we fetching data in filters. We need to be consistent.

Reply:
@anuveyatsu , the data from filters are used by `@apollo/client`.

But we can also use fetch... 

I update on this commit 7c63c3689d419ece1d6b280697b1d19eedb04a1b